back/engines/free/email_campaigns/app/serializers/email_campaigns/web_api/v1/campaign_serializer.rb

```diff
@@ -3,7 +3,7 @@
 module EmailCampaigns
   class WebApi::V1::CampaignSerializer < ::WebApi::V1::BaseSerializer
     extend GroupOrderingHelper
-    attributes :created_at, :updated_at
+    attributes :created_at, :updated_at, :resource_id
```

Comment:
Worth including it? is it used by the front-end?

Reply:
Hmm, maybe not yet. 🤔 Good point.

I imagine it will be used, in future, when we have other campaigns that use the `resource_id`, but then we can easily add it when it's needed.